helm/templates/env.yaml

```diff
@@ -12,7 +12,7 @@ data:
   {{- if .Values.debug }}
   LOG_LEVEL: DEBUG
   {{- else }}
-  LOG_LEVEL: INFO
+  LOG_LEVEL: {{.Values.defaultLogLevel}}
```

Comment:
```suggestion
  LOG_LEVEL: {{.Values.logLevel | default "INFO"}}
```

Reply:
I'd suggest to have an explicit default at this point to avoid having to specify `logLevel` in the `helm/values.yaml`.

Reply:
Hi @annekebr. Thanks for your review!

In an helm chart, the values.yaml file represents the API for consumer to know what parameters are supported.
For that reason, I suggest to keep and define the `logLevel` yaml key in the values.yaml file, with its default `INFO` value. This is a good place as well for users to know what is the default value just by reading the values.yaml file.

Reply:
Hi @sf-jmarcou. I agree the `helm/values.yaml` should be self-explanatory. I edited my suggestion below to reflect the default value in a comment.

We really want to keep our `helm/values.yaml` as minimal as possible and make additional configuration options optional. :)